### DISCUSSION TITLE
Sounds work perfectly in Visual Studio but stop when I install the application on Windows

### DISCUSSION THREAD

**Discussion by arianna-zanetti** [naudio/NAudio#1133] [Q&A]
Hi, I am trying to use Naudio in a Windows application. I am using the MixingSampleProvider to play sounds in the background, I followed this example: https://markheath.net/post/fire-and-forget-audio-playback-with. Everything works fine when I debug in Visual Studio, I have several cached sounds, and I tried both .wav files and .mp3 and they work. As soon as I install the application on my Windows PC, though, I can't hear any of the sounds. Has anyone else had a similar issue? Thank you.

**Comment by markheath** [comment]
Hard to say what exactly is causing this. Are you sure the audio files are available to the installed application? Are you getting any error messages. Make sure you subscribe to PlaybackStopped and log its exception. Are you successfully getting as far as initializing and starting playback?
